src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        ColorSupplier colorSupplier = new ColorSupplier();
+        String color1 = colorSupplier.getRandomColor().toString();
+        int number1 = getRandomNumber();
+        Ball ball = new Ball();
+        ball.setColor(color1);
+        ball.setNumber(number1);
+        return ball;
+    }
+
+    public int getRandomNumber() {
+        return new Random().nextInt(100);
```

Comment:
![image](https://github.com/mate-academy/jv-lottery/assets/63095184/08390f50-5dc6-49d6-aa4f-cc0adafd2fe7)

Reply:
not fixed